paddle/fluid/inference/tests/api/analyzer_int8_image_classification_tester.cc

```diff
@@ -28,6 +28,7 @@ void SetConfig(AnalysisConfig *cfg) {
   cfg->SwitchSpecifyInputNames();
   cfg->SetCpuMathLibraryNumThreads(FLAGS_paddle_num_threads);
   cfg->EnableMKLDNN();
+  cfg->pass_builder()->AppendPass("fc_mkldnn_pass");
```

Comment:
Why not add the pass to the suite of MKL-DNN passes?

Reply:
Please see https://github.com/PaddlePaddle/Paddle/pull/15226#pullrequestreview-238328905
Since MKL-DNN's FC was only bringing performance improvement on some topologies, it has been decided to have it enabled optionally.

Reply:
OK